idl/transaction.x

```diff
@@ -101,25 +101,35 @@ namespace mazzaroth
     Action action;
   };
 
+  // Batch of multiple transactions used by consensus
+  struct TransactionsBatch {
+    Transaction transactions<>;
+  };
 
-  // A transaction that has been executed, contains a receipt, and is
-  // ready to be stored in the ledger.
+  // A transaction batch that has been committed through consensus
   struct CommittedTransaction
   {
-    // The transaction itself
-    Transaction transaction;
+    // The batch that was committed
+    TransactionsBatch batch;
```

Comment:
Whats the point of this struct?
Why not just make this a var array of transactions?

Can you add a comment that the SignatureSet's payload is the commit.

Reply:
The reason the batch of transactions is a struct is because that is the object that is hashed to get the batchId used in the Consensus Commit message and we do not want to change that.

Yes I will add the comment.